plugins/go-build/bin/goenv-install

```diff
@@ -2,7 +2,7 @@
 #
 # Summary: Install a Go version using go-build
 #
-# Usage: goenv install [-f] [-kvp] <version>
+# Usage: goenv install [-f] [-kvp] <version>|latest|unstable
```

Comment:
Oh, I think this is a great approach actually.
This way both kind of developer are pleased.